obal/data/modules/copr_chroot.py

```diff
@@ -27,23 +28,19 @@ def main():
     external_repos = module.params['external_repos']
     buildroot_packages = module.params['buildroot_packages']
     modules = module.params['modules']
+    config_file = module.params['config_file']
 
     command = [
         'edit-chroot',
         "{}/{}".format(full_name(user, project), chroot)
     ]
 
-    if external_repos:
-        command.extend(['--repos', ' '.join(external_repos)])
-
-    if buildroot_packages:
-        command.extend(['--packages', ' '.join(buildroot_packages)])
-
-    if modules:
-        command.extend(['--modules', ','.join(modules)])
+    command.extend(['--repos', ' '.join(external_repos)])
+    command.extend(['--packages', ' '.join(buildroot_packages)])
+    command.extend(['--modules', ','.join(modules)])
```

Comment:
the defaults for those params are `[]`, so if the user didn't pass anything, these lines will expand to:
```
command.extend(['--repos', ''])
command.extend(['--packages', ''])
command.extend(['--modules', ''])
```

is that what you intended here?